schema/base/infrastructure.yaml

```diff
@@ -0,0 +1,103 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Infrastructure Schema
+description: TODO
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Point.json
+      - "$ref": https://geojson.org/schema/LineString.json
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ./defs.yaml#/$defs/propertyContainers/osmPropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+      - class
+    properties:
+      subType:
+        description: Further description of the type of infrastructure.
+        type: string
+        enum:
+          - airport
+          - bridge
+          - communication
+          - dam
+          - manhole
+          - pier
+          - power
+          - tower
+          - transit
+      class:
+        description: Further classification of the infrastructure type
+        type: string
+        enum:
```

Comment:
We should sort this list! Makes it a lot easier to grok it as a human...

Reply:
Agree! Sorry, you caught me in the middle of updating that...